ui/components/component-library/icon/README.mdx

```diff
@@ -94,7 +94,71 @@ import { Icon, ICON_NAMES } from '../../components/component-library';
 <Icon name={ICON_NAMES.ADD_SQUARE} color={IconColor.warningDefault} />
 <Icon name={ICON_NAMES.ADD_SQUARE} color={IconColor.warningInverse} />
 <Icon name={ICON_NAMES.ADD_SQUARE} color={IconColor.infoDefault} />
-<Icon name={ICON_NAMES.ADD_SQUARE} color={IconColor.infoInvers} />
+<Icon name={ICON_NAMES.ADD_SQUARE} color={IconColor.infoInverse} />
```

Comment:
Line 90, we need to update 

`<Icon name={ICON_NAMES.ADD_SQUARE} color={IconColor.infoInverse} />`

Reply:
Nice catch! Resolved https://github.com/MetaMask/metamask-extension/pull/17612/commits/4d564b76ab4f3142869f1817c19095fb68b3b946